Lib/platform.py

```diff
@@ -219,7 +220,10 @@ def libc_ver(executable=None, lib='', version='', chunksize=16384):
                     continue
                 if not m:
                     break
-            libcinit, glibc, glibcversion, so, threads, soversion, musl, muslversion = [
+            (
+                libcinit, glibc, glibcversion, so, threads, soversion,
+                musl, muslversion, musl_so, musl_sover
+            ) = [
                 s.decode('latin1') if s is not None else s
                 for s in m.groups()]
```

Comment:
At this point it makes sense to introduce a temporary variable before unpacking:
```suggestion
            decoded_groups = [s.decode('latin1') if s is not None else s
                              for s in m.groups()]
            (libcinit, glibc, glibcversion, so, threads, soversion,
             musl, muslversion, musl_so, musl_sover) = decoded_groups 
```